test/main/test.js

```diff
@@ -7,6 +7,10 @@ QUnit.test( "expect query and multiple issue", function( assert ) {
 	assert.ok( true );
 });
 
+QUnit.test( "expected of QUnit.test is deprecated", 42, function( assert ) {
+	assert.expect( 0 );
+});
+
```

Comment:
This test triggers a false positive for the changes coverage. We need to actual test if it's warning anything. That's where having a warning details on the log might help.